extras_install.sh

```diff
@@ -43,11 +47,15 @@ if [ ${ID} = "mendel" ]; then
 
 fi
 
+if [ "${os_name}" = "Ubuntu" ] && [ "${os_version}" = "20.04" ]; then
+   python3 -m pip  install tflite_runtime==2.8.0
```

Comment:
remove extra space b/w pip and install. Also add some comment about different version of tflite_runtime for different platforms.